Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/ui.py

```diff
@@ -46,6 +108,72 @@ def execute(self, context):
         """
         webbrowser.open(constants.WIKI_URL)
         return{'FINISHED'}
+class CustomProjectPath(bpy.types.Operator, ImportHelper):
+    """!
+    This Class is for setting a custom project path
+    """
+    bl_idname = "project.open_filebrowser"
+    bl_label = "Project Assets Folder"
+    bl_options = {'PRESET', 'UNDO'}
+
+    filter_glob: StringProperty(
+        default="//",
+        options={'HIDDEN'},
+        subtype='NONE',
+        maxlen=255,  # Max internal buffer length, longer would be clamped.
+        )
+
+    def execute(self, context):
+        # Look at current project list, if path not in list add it
+        real_path =  Path(self.filepath)
+        if real_path.is_dir():
+            bpy.types.Scene.selected_o3de_project_path = str(Path(self.filepath))
+        else:
+            bpy.types.Scene.selected_o3de_project_path = str(Path(self.filepath).parent)
+        if bpy.types.Scene.selected_o3de_project_path not in context.scene.o3de_projects_list:
+            o3de_utils.save_project_list_json(str(bpy.types.Scene.selected_o3de_project_path))
+            # Refesh the addon
+            addon_utils.enable('SceneExporter')
+            # Rebuild the project list
+            o3de_utils.build_projects_list() 
+        return {'FINISHED'}
+
+class AddColliderMesh(bpy.types.Operator):
+    """!
+    This Class is for the UI Wiki Button
+    """
+    bl_idname = "vin.collider"
+    bl_label = "Add a PhysX Collider mesh to selected."
+
+    def execute(self, context):
+        """!
+        This function will create the collision mesh.
+        """
+        # Create a Pop-Up confirm window
+        bpy.types.Scene.pop_up_confirm_label = 'Adding UDP Type: o3de_atom_phys to mesh.'
+        bpy.types.Scene.pop_up_question_label = 'Add UPD to a Duplicate mesh?'
+        bpy.types.Scene.udp_type = constants.UPD.get('o3de_atom_phys')
+        bpy.ops.message_confirm.popup('INVOKE_DEFAULT')
+        return{'FINISHED'}
+
+class AddLODMesh(bpy.types.Operator):
+    """!
+    This Class is for the UI Wiki Button
+    """
+    bl_idname = "vin.lod"
+    bl_label = "Add a LOD mesh to selected."
+
+    def execute(self, context):
+        """!
+        This function will create the LOD mesh
+        """
+        # Create a Pop-Up confirm window
+        bpy.types.Scene.pop_up_confirm_label = 'Adding UDP Type: o3de_atom_lod to mesh.'
+        bpy.types.Scene.pop_up_question_label = 'Add UPD to a Duplicate mesh?'
+        bpy.types.Scene.udp_type = constants.UPD.get('o3de_atom_lod')
+        bpy.ops.message_confirm.popup('INVOKE_DEFAULT')
+        return{'FINISHED'}

```

Comment:
I'm wondering if this is a convention I'm unaware of with Blender Python, but I'm curious why you return what looks to be a dictionary with a key and no value assigned? Couldn't this be just a string? Or does the caller require a dictionary formatted return?

Reply:
Yeah, this is just a value string that is called, I sort of set it up the groundwork to in the future have even more added to this dictionary that just 1 default variable, like a bunch of defaults for the UDP. But, not just on this iteration.